<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSettings</class>
 <widget class="QDialog" name="DlgSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="GroupSize">
     <property name="title">
      <string>Box dimensions</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="LabelWidth">
          <property name="text">
           <string>Width:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="SpinBoxWidth">
          <property name="minimum">
           <number>20</number>
          </property>
          <property name="maximum">
           <number>250</number>
          </property>
          <property name="value">
           <number>120</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="HSpacer3">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="LabelEmptyLines">
          <property name="text">
           <string>Empty lines:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="SpinBoxEmptyLines">
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="HSpacer4">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupDerocation">
     <property name="title">
      <string>Decoration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="1">
       <widget class="QLineEdit" name="EditBottomLeft">
        <property name="text">
         <string> *</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="LabelMiddleLeft">
        <property name="text">
         <string>Middle left:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QLineEdit" name="EditBottomCenter">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="5" column="7">
       <widget class="QLabel" name="LabelBottomRight">
        <property name="text">
         <string>Bottom right:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QLineEdit" name="EditTopCenter">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="LabelBottomLeft">
        <property name="text">
         <string>Bottom left:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="3" colspan="2">
       <widget class="QLabel" name="LabelBottomCenter">
        <property name="text">
         <string>Bottom center:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="EditMiddleLeft">
        <property name="text">
         <string> *</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="LabelTopLeft">
        <property name="text">
         <string>Top left:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <spacer name="HSpacer1">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="8">
       <widget class="QLineEdit" name="EditMiddleRight">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="QLabel" name="LabelTopRight">
        <property name="text">
         <string>Top right:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="7">
       <widget class="QLabel" name="LabelMiddleRight">
        <property name="text">
         <string>Middle right:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="8">
       <widget class="QLineEdit" name="EditTopRight">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="EditTopLeft">
        <property name="text">
         <string>/*</string>
        </property>
       </widget>
      </item>
      <item row="5" column="8">
       <widget class="QLineEdit" name="EditBottomRight">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <spacer name="HSpacer2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="3" colspan="2">
       <widget class="QLabel" name="LabelTopCenter">
        <property name="text">
         <string>Top center:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Clipboard</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="CheckAutoCopy">
        <property name="text">
         <string>Automatically copy output in the clipboard</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="HSpacer5">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="LabelKeySequenceCopy">
        <property name="text">
         <string>Copy to clipboard:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QKeySequenceEdit" name="KeySequenceCopy"/>
      </item>
      <item>
       <spacer name="HSpacer6">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>156</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLayoutButtons">
     <item>
      <spacer name="HSpacerButtons">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>EditTopLeft</tabstop>
  <tabstop>EditTopCenter</tabstop>
  <tabstop>EditTopRight</tabstop>
  <tabstop>EditMiddleLeft</tabstop>
  <tabstop>EditMiddleRight</tabstop>
  <tabstop>EditBottomLeft</tabstop>
  <tabstop>EditBottomCenter</tabstop>
  <tabstop>EditBottomRight</tabstop>
  <tabstop>SpinBoxEmptyLines</tabstop>
  <tabstop>ButtonOK</tabstop>
  <tabstop>ButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
